.aws/src/config/index.ts

```diff
@@ -15,6 +15,12 @@ const branch = isDev ? 'dev' : 'main';
 const cacheNodes = isDev ? 2 : 2;
 const cacheSize = isDev ? 'cache.t2.micro' : 'cache.t3.medium';
 
+// const snowplowEndpoint = isDev
+//   ? 'com-getpocket-prod1.mini.snplow.net'
+//   : 'com-getpocket-prod1.collector.snplow.net';
+//todo: send to prod after disconnecting existing data from user-api
+const snowplowEndpoint = 'com-getpocket-prod1.mini.snplow.net'
```

Comment:
is there a ticket to track this?

Reply:
nope, created one https://getpocket.atlassian.net/browse/INFRA-766